CHANGELOG.md

```diff
@@ -4,6 +4,10 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [3.0.3] - 2020-10-15
```

Comment:
this is a breaking change, not a patch. My client code has broken because `->toArray()` returns array of arrays now, not array of entities. Moreover, docblock is not correct now - it says that return type is Entity[], but it is array of arrays.

Reply:
https://github.com/DoclerLabs/api-client-generator/pull/16